tests/test_events_api.py

```diff
@@ -0,0 +1,137 @@
+from os import environ
+from typing import Optional
+from unittest.mock import MagicMock, patch
+
+from lbz.aws_boto3 import Boto3Client
+from lbz.events.api import BaseEvent, EventAPI
+
+
+class MyTestEvent(BaseEvent):
+    type = "MY_TEST_EVENT"
+
+
+class TestBaseNewEvent:
+    def test_base_new_event(self) -> None:
+        new_event = MyTestEvent({"x": 1})
+
+        assert hasattr(new_event, "raw_data")
+        assert hasattr(new_event, "data")
+        assert new_event.type == "MY_TEST_EVENT"
+        assert new_event.raw_data == {"x": 1}
+        assert new_event.data == '{"x": 1}'
+
+
+class TestEventApi:
+    event_api: Optional[EventAPI] = None
```

Comment:
Why do you need that here?

Reply:
mypy

Reply:
But you removed this line anyway, so it stopped being a problem?